### DISCUSSION TITLE
Error 500

### DISCUSSION THREAD

**Discussion by ghost** [mecha-cms/mecha#100] [Q&A]
Saya coba menambahkan pass extention ke folder x, lalu menambahkan passwords di YAML, dan saya coba akses laman kok jadi error 500 ya?

Lalu ada kasus ketika saya mencoba submit sesuatu di panel, misalnya add user, terjadi masalah yakni, peramban tidak ngebaca css, tetapi data tetap masuk ke folder user.

Apakah yang menjadi masalah ialah bahwa saya mengunduh folder di situs mecha-cms.com? Apakah kode-kode yang saya unduh di situs Mecha di sana berbeda dengan kode yang di Github? Untuk mengetahui pembaharuan ekstensi gimana ya Mas? Saya bingung kode Mecha yang paling mutakhir itu yang di Github atau di website Mecha.

**Comment by taufik-nurrohman** [comment]
Kalau ekstensi _pass_ dilepas bagaimana? Ada bedanya?

Oya, ekstensi _pass_ agaknya sudah tidak kompatibel dengan versi sekarang karena lama tidak Saya perbarui. Kalau sudah rilis versi stabil berikutnya mau coba Saya perbarui :+1:

> Lalu ada kasus ketika saya mencoba submit sesuatu di panel, misalnya add user, terjadi masalah yakni, peramban tidak ngebaca css, tetapi data tetap masuk ke folder user.

Coba buka `.\lot\x\panel\state.php` lalu ubah data konfigurasi `'fetch'` dari `true` menjadi `false`. Lalu buka `.\index.php`, ubah nilai konstanta `DEBUG` menjadi `true`. Segarkan laman, kemudian coba submit lagi. Kasih tahu error yang muncul ke Saya seperti apa. Biasanya itu cuma karena halaman error 404.
